src/packer.pkr.hcl

```diff
@@ -131,8 +131,15 @@ build {
   }
 
   provisioner "shell" {
-    execute_command = "chmod +x {{ .Path }}; sudo env {{ .Vars }} {{ .Path }} ; rm -f {{ .Path }}"
+    execute_command = "{{ .Vars }} bash '{{ .Path }}'"
```

Comment:
`chmod +x {{ .Path }}; sudo env and ; rm -f {{ .Path }}` is not removed - it's in the next provisioner block.
see: https://github.com/cisagov/ubuntu-server-packer/pull/15#discussion_r1320286347

As for the single quotes - honestly, I don't know. I can remove them and try without. I was grabbing that code as a solution (can't find it now). I'll remove the quotes and we can see if we run into any issue. I don't expect we will.

Reply:
See [433052a](https://github.com/cisagov/ubuntu-server-packer/pull/15/commits/433052a88f2b366b2623e380775ba65c72c0c0d5)

Reply:
Found the article - I was implementing this solution, and `{{ .Path }}` was single-quoted here:
https://discuss.hashicorp.com/t/permission-denied-on-shell-script-provisioner/22863

Reply:
@jsf9k - I removed the single quotes. I just included the link above to explain the source of the line of code. The big thing I had to do was call bash to run it, because we couldn't execute the code directly from `/tmp` due to having the `noexec` bit.

Reply:
Yes, that is exactly what I was asking about.  I'm glad my jumbled words finally made some sense.